lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -467,14 +491,20 @@ namespace Intl
         return ret;
     }
 
-    bool IsLocaleAvailable(_In_z_ const char *locale)
+    bool IsLocaleAvailable(_In_z_ const char *langtag)
     {
         int32_t countAvailable = uloc_countAvailable();
         Assert(countAvailable > 0);
+
+        UErrorCode status = U_ZERO_ERROR;
+        char localeID[ULOC_FULLNAME_CAPACITY] = { 0 };
+        int32_t length = 0;
+        uloc_forLanguageTag(langtag, localeID, ULOC_FULLNAME_CAPACITY, &length, &status);
```

Comment:
>uloc_forLanguageTag(langtag, localeID [](start = 8, length = 37)

comment explaining that this is converting langtag to localeID (the call doesn't make this purpose clear)